src/manifest/dbtProject.ts

```diff
@@ -308,10 +312,30 @@ export class DBTProject implements Disposable {
       await this.python?.lock(
         (python) => python!`to_dict(project.safe_parse_project())`,
       );
+      this.diagnostics.rebuildManifestDiagnostics.clear();
     } catch (exc) {
-      this.telemetry.sendTelemetryError("pythonBridgeCannotParseProject", exc, {
-        adapter: this.adapterType,
-      });
+      if (exc instanceof PythonException) {
+        this.diagnostics.rebuildManifestDiagnostics.set(
+          Uri.joinPath(this.projectRoot, DBTProject.DBT_PROJECT_FILE),
+          [new Diagnostic(new Range(0, 0, 999, 999), exc.exception.message)],
```

Comment:
It's not clear where the range is coming from. Is it to limit the message size? If yes, I am not sure if the range would capture the exact problem within it.  Also, adding a comment explaining the rationale behind the range will be helpful for posterity purposes.

Reply:
WE need to associate it with a file but that is hard in our case, so we take the project config.

I will add the comment.